packages/lesswrong/components/posts/AllPostsPage.jsx

```diff
@@ -90,12 +91,13 @@ class AllPostsPage extends Component {
         timeframe={currentTimeframe}
         postListParameters={{
           view: 'timeframe',
+          limit: parseInt(query.limit),
```

Comment:
This should handle the case where `query.limit` is undefined or not a number. I'm not sure what passing `NaN` here will do, but it's likely to be trouble.